client/src/constants/participantTableConstants.js

```diff
@@ -157,7 +157,24 @@ const columns = {
   phoneNumber: { id: 'phoneNumber', name: 'Phone Number', sortOrder: 8 },
   emailAddress: { id: 'emailAddress', name: 'Email Address', sortOrder: 9 },
   preferredLocation: { id: 'preferredLocation', name: 'Preferred Region(s)', sortOrder: 10 },
+  driverLicense: { id: 'driverLicense', name: `Driver's License`, sortOrder: 10.1 },
```

Comment:
how come these ones are decimals?

Reply:
`sortOrder` is a comparator parameter to sort columns, so it doesn't need to be an integer.
I just didn't want to reset all the higher ones.